index.html

```diff
@@ -92,8 +94,154 @@ <h2 class="card__question">Woher kommt der Name "Frisbee"?</h2>
+        ></button>
+        <h2 class="card__question">Woher kommt der Name "Frisbee"?</h2>
+        <button class="card__button">Antwort</button>
+        <p class="card__answer card__answer--hidden">
+          Der Name geht auf die Bäckerei "Frisbie" in Conneticut zurück. Deren
+          kreisrunde Backformen aus Zinn waren die Vorläufer der fliegenden
+          Plastikscheiben.
+        </p>
+        <ul class="tags">
+          <li class="tags__item">Sport</li>
+          <li class="tags__item">Freizeit</li>
+          <li class="tags__item">Name</li>
+          <li class="tags__item">Geschichte</li>
+        </ul>
+      </section>
+    </main>
+
+    <main class="main main--hidden" id="create">
+      <section class="card__create">
+        <form>
+          <br />
+          <label for="question">Deine Frage</label><br />
+          <textarea id="question" name="question" rows="3" cols="32">
+Hier deine Frage stellen
+          </textarea>
+          <br />
+          <label for="answer">Deine Antwort</label><br />
+          <textarea id="answer" name="answer" rows="3" cols="32">
+Hier deine Antwort geben
+          </textarea>
+          <br />
+          <label for="tags">Schlagworte</label><br />
+          <textarea id="tags" name="tags" rows="3" cols="32">
+Tags durch Komma trennen
+          </textarea>
+          <br />
+          <input type="submit" value="Submit" />
+        </form>
+      </section>
+    </main>
+
+    <main class="main main--hidden" id="profile">
+      <section class="account__profile">
+        <img
+          class="account__image"
+          src="/img/account/drkatze.jpeg"
+          alt="Profilbild"
+        />
+
+        <h2 class="account__name">Doktor Katze</h2>
+        <div class="account__about">
+          <h3 class="account__about-title">Über mich</h3>
+        </div>
+        <article class="account__about_text">
+          Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Aenean
+          commodo ligula eget dolor. Aenean massa. Cum sociis natoque...
+        </article>
+        <div>
+          <h3 class="account__skills">Skills</h3>
+          <ul class="account__skills_tags">
+            <li class="account__skills_tag">HTML</li>
+            <li class="account__skills_tag">CSS</li>
+            <li class="account__skills_tag">Java</li>
+            <li class="account__skills_tag">PHP</li>
+          </ul>
+        </div>
+        <div class="account__button">
+          <button class="account__botton_logout">abmelden</button>
+        </div>
+      </section>
+    </main>
+
     <footer>
-      <nav>
+      <nav class="nav">
+        <button class="nav__link nav__link--active" id="question-button">
+          Home
+        </button>
+        <button class="nav__link" id="bookmark-button">Bookmark</button>
+        <button class="nav__link" id="create-button">Create</button>
+        <button class="nav__link" id="profile-button">Profile</button>
+
+        <!----- 
         <ul>
```

Comment:
You don't need this anymore so please delete it! 🙂 We usually like to have our main branch as clean as possible with as few comments as possible.

Reply:
Already deleted 🗑 🤓